flood_adapt/object_model/hazard/hazard.py

```diff
@@ -144,6 +134,23 @@ def set_simulation_paths(self) -> None:
                     )
                 )
 
+    def has_run_check(self) -> bool:
+        """_summary_
+
+        Returns
+        -------
+        bool
+            _description_
+        """
+        self._get_flood_map_path()
+
+        has_run = False
+
+        if self.flood_map_path:
+            has_run = True
+
+        return has_run
```

Comment:
This is fine for now, but I feel there must be a better way to find out if a model has been run

Reply:
I updated the method to check explicitly for file existense